test/sessionClient.spec.js

```diff
@@ -74,6 +74,43 @@ describe('Session Client', () => {
 				});
 		});
 
+
+		it('Should get user login info for a valid secure session when kat2fa flag is on', done => {
+			if (mockAPI) {
```

Comment:
I know we have this for the other tests, but we shouldn't really have it at all, as we don't run our tests against an actual API, it will always be mocked.
Same goes for the other tests.